sessionctx/variable/sysvar.go

```diff
@@ -683,6 +685,13 @@ var defaultSysVars = []*SysVar{
 	}, GetGlobal: func(s *SessionVars) (string, error) {
 		return fmt.Sprint(QueryLogMaxLen.Load()), nil
 	}},
+	{Scope: ScopeGlobal, Name: TiDBCommitterConcurrency, Value: strconv.Itoa(DefTiDBCommitterConcurrency), Type: TypeInt, MinValue: 1, MaxValue: 10000, SetGlobal: func(s *SessionVars, val string) error {
+		cfg := config.GetGlobalConfig().GetTiKVConfig()
```

Comment:
Sorry, I found a bug. It needs to update the atomic value first:
```suggestion
	{Scope: ScopeGlobal, Name: TiDBCommitterConcurrency, Value: strconv.Itoa(DefTiDBCommitterConcurrency), Type: TypeInt, MinValue: 1, MaxValue: 10000, SetGlobal: func(s *SessionVars, val string) error {
		tikvutil.CommitterConcurrency.Store(..)
		cfg := config.GetGlobalConfig().GetTiKVConfig()
```